### DISCUSSION TITLE
[doc] single is missing

### DISCUSSION THREAD

**Discussion by pierre-H** [directus/directus#5601] [Archive]
Hello,

In the [documentation](https://docs.directus.io/reference/api/query/), the parameter `single` is missing.
But, it does exist in the TypeScript definition : https://github.com/directus/directus/blob/4366194ac30af49a8be3bb704753e21ae83663aa/api/src/types/query.ts#L10

Is it normal ?

**Comment by rijkvanzanten** [comment]
Yes. `single` is deprecated and not meant for public use
